### PR TITLE
Convince your boss

### PR DESCRIPTION
**Pull request by necccc**

### CODE REVIEW

File: pages/convince-your-boss.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import ColoredSection from "../ui/colored-section";
+import SocialShare from "../ui/social-share";
+
+function CallForPapers() {
+  return (
+    <>
+      <SocialShare
+        title="Convince your boss!"
+        description=""
+        socialShareImage="social-share/convince-your-boss.png"
```
Comment (greg-nagy) [inline pages/convince-your-boss.tsx:13]: "Did you mean to add an attachment?" - displays in gmail popup :)

File: pages/convince-your-boss.tsx [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+      />
+      <Divider>
+        <h1>Convincing Your Employer to Invest in Your Development</h1>
```
Comment (greg-nagy) [inline pages/convince-your-boss.tsx:16]: This is a long title and does not look good on most of the resolutions

Works on 1700+px and 1240-1280px

**Review by greg-nagy — APPROVED** [review]
Content looks good, left 2 comments that might be good to fix before release but non blockers

Based on the landing page section does it mean that we should not merge this before 2nd of April?

File: pages/convince-your-boss.tsx (hunk 1 above)
Comment (necccc) [reply pages/convince-your-boss.tsx:12]: fixed!

File: pages/convince-your-boss.tsx (hunk 2 above)
Comment (necccc) [reply pages/convince-your-boss.tsx]: fixed!

**Pull request merged**